src/Cooked/MockChain/GenerateTx/Body.hs

```diff
@@ -89,7 +88,7 @@ txSkelToBodyContent skel@TxSkel {..} | txSkelReferenceInputs <- txSkelReferenceT
       txCertificates = Cardano.TxCertificatesNone -- That's what plutus-apps does as well
       txUpdateProposal = Cardano.TxUpdateProposalNone -- That's what plutus-apps does as well
       txScriptValidity = Cardano.TxScriptValidityNone -- That's what plutus-apps does as well
-      txVotingProcedures = Nothing -- TODO, same as above
+      txVotingProcedures = Nothing
```

Comment:
Out of curiosity, what was this deprecated TODO about?

Reply:
The TODO refers to the future implementation of voting procedures and overall of the body fields that we don't yet support.